src/main/java/net/objecthunter/exp4j/function/Functions.java

```diff
@@ -183,6 +186,24 @@ public double apply(double... args) {
                 }
             }
         };
+        builtinFunctions[INDEX_SIN_DEGREES] = new Function("Sin") {
+            @Override
+            public double apply(double... args) {
+                return Math.sin(Math.toRadians(args[0]));
```

Comment:
Maybe instead of adding three new functions, namely: ```Sin```, ```Cos``` and ```Tan```, simply add one new function, i.e. ```rad(..)``` to convert from degrees to radians. By doing this you can reduce number of functions and use classic ```sin```, ```cos```, ```tan``` which are build in, e.g. ```sin(rad(a + b) / 2)```.